dispatch.go

```diff
@@ -166,11 +177,15 @@ func (d *dispatch) Call(ctx context.Context, name string, arg interface{}, res i
 }
 
 func (d *dispatch) Notify(ctx context.Context, name string, arg interface{}) error {
-	err := d.dispatchMessage(ctx, MethodNotify, name, arg)
-	if err != nil {
-		return err
-	}
-	d.log.ClientNotify(name, arg)
+	errCh := d.dispatchMessage(ctx, MethodNotify, name, arg)
+	go func() {
```

Comment:
We might not right now, but going forward we may wish to make `Notify` behave more like calls, with sequences, allowing us to cancel in-flight notifications.